setup/hr_holidays_previous_type/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```

Comment:
is it correct to have this file in there? .eggs is in the .gitignore file of the repository: https://github.com/OCA/hr/blob/bd2a36506d89d747145048d9287bba491c5000d5/.gitignore#L24